tools/debian_buildenv.sh

```diff
@@ -27,6 +27,12 @@ case "$COMMAND" in
         esac
 
         sudo apt-get update
+        if [ $(dpkg-query -W -f='${Status}' jackd2 2>/dev/null | grep -c "ok installed") -eq 1 ];
```

Comment:
Please add a comment explaining this. It is not obvious.

Reply:
If jackd2 is installed, as per the dpkg database, then, install libjack-jackd2-dev